spec/controllers/spree/users_controller_spec.rb

```diff
@@ -83,8 +83,9 @@
     it 'should create a new user' do
       post :create,
            params: { user: { email: 'foobar@example.com', password: 'foobar123',
-                             password_confirmation: 'foobar123' } }
+                             password_confirmation: 'foobar123', locale: 'es' } }
       expect(assigns[:user].new_record?).to be_falsey
+      expect(assigns[:user].reload.locale).to eq('es')
```

Comment:
This seems to fail.

Reply:
Noticed that committed by mistake, so removed it in 07353de7870a5321ab26d30df57e1861b67ffe86. 
The goal was to test admin controller, but this is outside of admin areas :) 

Because changed `::PermittedAttributes::User` in 07353de7870a5321ab26d30df57e1861b67ffe86, then locally the test was passing, but removed it because it was not intentional, neither that view was updated.